src/generated/resources/data/trutermproject/neoforge/biome_modifier/add_beryl_ore.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't have a lot to say about the `json` files as I don't fully have context for how they work.